backend/main.py

```diff
@@ -155,22 +152,13 @@ async def retrieve_current_track_information():
 
 
 @fast_app.put("/stop-spotihue")
-async def stop_spotihue(lights: List[str] = None):
+async def stop_spotihue():
     try:
-        if not lights:
```

Comment:
Is this because we don't need the lights to kill the process? Should we have an endpoint that gets hit for turning the lights back to the 'normal' color? That's what I think should happen if anything happens to spotihue... Kill the process and change the lights back to 'normal' color.

Reply:
Yeah, we don't need the lights as request input here to kill the task. We'll need them from somewhere if we want to change light colors outside of the run_spotihue task, but we should be able to accomplish that by inspecting the Task object's parameters. I think it'll be safer too in terms of making sure that the lights changed are the correct ones - if we do end up needing to cache the lights, I think it would be better for the celery task to do that caching (since we'll only ever have 1 task running), just to be sure that the stored lights are never out of sync with the running task's lights...